### DISCUSSION TITLE
How to scan dot-files?

### DISCUSSION THREAD

**Discussion by JerryBian** [photoview/photoview#673] [Q&A and Troubleshooting]
Just started to use `photoview`, looks like the scanner cannot detect directories start with dot?

such as `.photo1/1.png`. Do we have environment varibale to enable it?

**Comment by JerryBian** [comment]
Hi, just want to know the dot file is not supported or anything I am not aware of?

**Reply by viktorstrate** [reply]
Photoview automatically ignore hidden directories and there is currently no way to turn that on. It would be a nice feature to have an environment variable or something like that though.

**Reply by JerryBian** [reply]
Thanks.
